### DISCUSSION TITLE
profiles

### DISCUSSION THREAD

**Discussion by wangqingpen** [edgexfoundry/community#202] [Q&A]
![image](https://github.com/edgexfoundry/community/assets/72652818/d032ebe1-05d4-4a5a-9674-6143ba9816d0)
I want to use the mask shift argument to rewrite or add code on which side of edgexfoudry-sdk-go

**Comment by jiekechoo** [comment]
@wangqingpen Please refer here: https://github.com/edgexfoundry/device-sdk-go/blob/7730e9b2488ebe82cf8f1ec58f41fa934430b51b/internal/transformer/transformresult.go#L354-L398

Jieke
YIQISOFT
